web/src/components/UploadAirgapBundle.jsx

```diff
@@ -584,11 +584,7 @@ class UploadAirgapBundle extends React.Component {
                     ) : supportBundleCommand ? (
                       <div>
                         <div className="u-marginTop--10">
-                          <a
-                            href="#"
-                            className="replicated-link"
-                            onClick={this.toggleShowRun}
-                          >
+                          <a href="#" onClick={this.toggleShowRun}>
```

Comment:
this doesn't need the `link` class anymore?

Reply:
@rianfowler not for this specific link.